.azure/hpu-tests.yml

```diff
@@ -45,6 +45,10 @@ jobs:
           image: "1.15.1/ubuntu22.04/habanalabs/pytorch-installer-2.2.0:latest"
           dependency: "lightning"
           pkg_source: "source"
+        'w. lightning | master':
```

Comment:
we have 2*4 parallel pipelines. Fifth one need to wait or we better pend on one pipeline to finish

Reply:
Fixed it using one of the existing pipeline.